### PR TITLE
Add aws China regions ECR support

### PR DESCRIPTION
**Pull request by liangfb**
Add aws China regions ECR support, both of cn-north-1 and cn-northwest-1 endpoints.

### CODE REVIEW

**Comment by bitsf** [comment]
@liangfb please rebase the pr, in order to make CI CodeQL run

File: src/portal/src/lib/components/create-edit-endpoint/create-edit-endpoint.component.spec.ts [hunk 1]
```diff
@@ -198,7 +198,16 @@ describe("CreateEditEndpointComponent (inline template)", () => {
...
-          }
+	  },
+	  {
+	    "key": "cn-north-1",
+	    "value": "https://api.ecr.cn-north-1.amazonaws.com.cn"
+	  },
+	  {
+	    "key": "cn-northwest-1",
+	    "value": "https://api.ecr.cn-northwest-1.amazonaws.com.cn"
+	  }
```
Comment (ywk253100) [inline src/portal/src/lib/components/create-edit-endpoint/create-edit-endpoint.component.spec.ts:209]: The changes for UI codes isn't necessary, UI should render the endpoints automatically if you change the backend codes.

**Comment by ywk253100** [comment]
@liangfb Please squash your commit into single one and pass the required checks

**Review by ywk253100 — APPROVED** [review]

**Comment by bitsf** [comment]
@liangfb your code failed the CI check
and only keep one commit in the PR, squash them

**Comment by bitsf** [comment]
@liangfb your code failed the CI check
and only keep one commit in the PR, squash them

**Pull request closed without merging**